e2e/src/test/groovy/gov/hhs/cdc/trustedintermediary/e2e/OrderTest.groovy

```diff
@@ -47,6 +47,7 @@ class OrderTest extends Specification {
         //test that everything else is the same except the MessageHeader's event
         parsedSentPayload.entry[0].resource.remove("eventCoding")
         parsedLabOrderJsonFile.entry[0].resource.remove("eventCoding")
+        parsedSentPayload.entry[24].resource.remove("contact")
```

Comment:
We should also check that we added the contact information to the `parsedSentPayload` earlier in this test.  Similar how we are testing that we correctly set the MessageHeader's event to an OML_O21.